ember-app/app/models/new/issue.js

```diff
@@ -187,7 +187,14 @@ var Issue = Model.extend({
       this.set("data.body_html", response.body_html);
       return this;
     }.bind(this), "json");
-  }
+  },
+  maxMinOrderFix: function(){
+    var order = this.get('order');
+    while(order < 1e-319){ order *= 10; }
+    while(order > 1e307){ order /= 10; }
+
+    this.set('order', order);
+  }.on('init')
```

Comment:
So the larger question then is if we should refresh the meta-data when the description has been edited. It only affects people who damper with their cards, but our users _are_ developers.